src/api-reference/request/request.markdown

```diff
@@ -39,11 +39,13 @@ Manage documents used for pre-spend authorizations within Concur Request.
 POST /v4/requests
 
 *Parameters:*
+
```

Comment:
The table structure seems overly complex and I'm not sure Jekyll will understand the complexity.

I suggest using this approach...

**Source**

```
Name|Parameter Type|Data Type|Description
----|--------------|---------|-----------
-|body|[Request](#request)|**Required** The content of the Request to create
```

**Result**

Name|Parameter Type|Data Type|Description
----|--------------|---------|-----------
-|body|[Request](#request)|**Required** The content of the Request to create